Troupon.Catalog.Api/Controllers/CatalogController.cs

```diff
@@ -18,35 +19,20 @@ namespace Troupon.Catalog.Api.Controllers
 {
   [ApiController]
   [Route("api/v{version:apiVersion}/[controller]")]
-  [ProducesResponseType(typeof(ValidationProblemDetails), StatusCodes.Status400BadRequest)]
-  [ProducesResponseType(StatusCodes.Status406NotAcceptable)]
-  [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status409Conflict)]
-  [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
-  [ProducesResponseType(StatusCodes.Status200OK)]
-  [ApiVersion("1.0")]
-  [ApiVersion("2.0")]
+  [ApiConventionType(typeof(PwcApiConventions))]
   public class CatalogController : BaseController
   {
     public CatalogController(IMapper mapper, IMediator mediator)
       : base(mediator, mapper)
     {
     }
 
-    /// <summary>
```

Comment:
We don't keep the documentation?

Reply:
Très bonne idée, mon commentaire visait surtout en démarrer une conversation générale sur ce qu'on veut faire en terme de documentation dans le code.

Reply:
Super ! Alors on va l'ajouter dans le backlog 👍